packages/framework/presence/src/datastorePresenceManagerFactory.ts

```diff
@@ -59,5 +93,9 @@ export const ExperimentalPresenceManager =
 export async function acquirePresenceViaDataObject(
 	fluidLoadable: ExperimentalPresenceDO,
 ): Promise<IPresence> {
-	throw new Error("Not implemented");
+	if (fluidLoadable instanceof PresenceManagerDataObject) {
+		return fluidLoadable.presenceManager();
```

Comment:
don't think this function needs to be async anymore

Reply:
I think you are correct - will get in a follow-up change :)